### DISCUSSION TITLE
split 1 long line into multiple by commas or dots

### DISCUSSION THREAD

**Discussion by Geniusssmit** [SubtitleEdit/subtitleedit#8368] [Q&A]
How can I split 1 long line into multiple by commas or dots. For example:
```
Zorian’s eyes abruptly shot open as a sharp pain erupted from his stomach. His whole body convulsed, buckling against the object that fell on him, and suddenly he was wide awake, not a trace of drowsiness in his mind.
```

into 5 lines

```
Zorian’s eyes abruptly shot open as a sharp pain erupted from his stomach. 
His whole body convulsed, 
buckling against the object that fell on him, 
and suddenly he was wide awake, 
not a trace of drowsiness in his mind.
```

**Comment by niksedk** [comment]
You can try File - Import - Plain text...

**Reply by Geniusssmit** [reply]
how do I import .srt file into it?

**Reply by niksedk** [reply]
Ahhhh, you have a .srt file...  

SE cannot do this exactly as you want, but try Tools - Break/split long lines...

**Comment by Purfview** [comment]
How did you got such long lines?

**Reply by Geniusssmit** [reply]
I used whisper to generate them. They do not end at commas etc, so I merged with "Merge short lines"

**Reply by Purfview** [reply]
No idea what you "merge" when your post is about how to "split"...

Then you can use whisper to generate what you want.

**Reply by Geniusssmit** [reply]
I had to merge first to have subs that ends with "," "?" "." etc. Whisper doesn't allow to change anything really. Is there a way to solve my problem with subtitleedit?

**Reply by Purfview** [reply]
Use Standalone Faster-Whisper, it has many options for such problems, for example `--standard` option.

**Comment by haytam776** [comment]
We really need that ! thank you
